vpr/src/draw/draw.cpp

```diff
@@ -2161,9 +2161,10 @@ void draw_get_rr_pin_coords(const t_rr_node& node, float* xcen, float* ycen, con
     float offset, xc, yc, step;
     t_physical_tile_type_ptr type;
     auto& device_ctx = g_vpr_ctx.device();
+    const auto& rr_graph = device_ctx.rr_graph;
 
-    i = node.xlow();
-    j = node.ylow();
+    i = rr_graph.node_xlow(node.id());
```

Comment:
I think we could shorten the code with
auto inode = node.id();
and then using inode below.